src/lib/css/mixins/_design.scss

```diff
@@ -45,24 +46,25 @@
 }
 
 @mixin disabled {
-  cursor: not-allowed;
-  box-shadow: none;
+  cursor: not-allowed !important;
```

Comment:
Is `!important` really necessary? Also for the others following.

Reply:
I think so, those styles are important and the specificity is pretty low so easily overwritten

Reply:
It's kind of a "last resort". Usually you should not use `!important`.